apps/daffio/src/app/content/pwa/pages/pwa/pwa.component.html

```diff
@@ -0,0 +1,159 @@
+        <img daffFeatureIcon src="/assets/why-pwa/integrated.svg" />
+        <h4 daffFeatureTitle>
+          Integrated
+        </h4>
+        <p daffFeatureSubtitle>
+          A user experience more integrated with the native platform &mdash; app icon for home screen access, push notifications, immersive full screen experience, and more.
+        </p>
+      </daff-feature>
+      <daff-feature mode="compact">
+        <img daffFeatureIcon src="/assets/why-pwa/reliable.svg" />
+        <h4 daffFeatureTitle>
+          Reliable
+        </h4>
+        <p daffFeatureSubtitle>
+          With the help of services workers, PWAs cache key assets and data on devices, allow applications to load and perform samlessly in slow to no connectivity.
+        </p>
+      </daff-feature>
+      <daff-feature mode="compact">
+        <img daffFeatureIcon src="/assets/why-pwa/engaging.svg" />
+        <h4 daffFeatureTitle>
+          Engaging
+        </h4>
+        <p daffFeatureSubtitle>
+          The ability to send web push notifications can help drive engagement and increase conversion rates by keeping users notified and up-to-date.
+        </p>
+      </daff-feature>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered" class="understanding">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      Understanding PWAs
+    </h3>
+    <p daffCalloutSubtitle>
+      Learn how Progressive Web Apps can enhance your presence on the web.
+    </p>
+    <div class="understanding__grid">
+      <a href="https://developers.google.com/web/showcase/vertical/#retail" target="_blank" rel="noopener noreferrer">
+        <daff-feature class="understanding__feature">
+          <img daffFeatureIcon src="/assets/why-pwa/book-icon.svg" />
+          <p daffFeatureSubheader>Google Developers</p>
+          <h4 daffFeatureTitle>
+            PWA Case Studies
+          </h4>
+          <p daffFeatureSubtitle>
+            Learn why and how developers have used the web to create amazing web experiences for their users.
+          </p>
+        </daff-feature>
+      </a>
+      <a href="https://www.pwastats.com/" target="_blank" rel="noopener noreferrer">
+        <daff-feature class="understanding__feature">
+          <img daffFeatureIcon src="/assets/why-pwa/stat-icon.svg" />
+          <p daffFeatureSubheader>PWA Stats</p>
+          <h4 daffFeatureTitle>
+            PWA Conversion Stats
+          </h4>
+          <p daffFeatureSubtitle>
+            A community-driven list of stats and news focused on Progressive Web Apps.
+          </p>
+        </daff-feature>
+      </a>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      Who's building PWAs?
+    </h3>
+    <p daffCalloutSubtitle>
+      Brands are increasingly utilizing Progressive Web Apps to bridge the gap between native apps and websites to create better user experiences.
```

Comment:
The double "ing" is throwing me off. I think "Brands increasingly utilize" is easier to read